package.json

```diff
@@ -6,6 +6,7 @@
     "bootstrap": "^4.3.1",
     "jquery": "^3.4.1",
     "popper.js": "^1.15.0",
+    "serialize-javascript": "^2.1.1",
```

Comment:
it looks like you added this dependency.  What does it do?

Reply:
I dunno;  we were getting a github alert, so i think it was a secondary dependency before, but not using latest version.

Reply:
appears to be an indirect dependency of `compression-webpack-plugin`:
https://github.com/sul-dlss/was-registrar-app/blob/ebbde0ff2008e741661f2d4eac98a2d285cd690c/yarn.lock#L1733-L1743

Reply:
and `terser-webpack-plugin`

https://github.com/sul-dlss/was-registrar-app/blob/ebbde0ff2008e741661f2d4eac98a2d285cd690c/yarn.lock#L6560-L6573